--- ToutDoux-1.2.7/configure.in.orig	2004-04-08 22:19:37.000000000 +0200
+++ ToutDoux-1.2.7/configure.in	2004-04-08 22:22:10.245657136 +0200
@@ -427,20 +427,10 @@
 core-plugins/Makefile
 data-plugins/Makefile
 filters/Makefile
-dnl scripts/Makefile
-dnl backends/Makefile
 
-dnl ***************************************************************************
-dnl fr: Script de compilation : Application
-dnl en: Compilation script : Application
-dnl ***************************************************************************
 
 app/Makefile
 
-dnl ***************************************************************************
-dnl fr: Script de compilation : Greffons noyaux
-dnl en: Compilation script : Core plug-ins
-dnl ***************************************************************************
 
 core-plugins/dialog_messages/Makefile
 core-plugins/tools_query/Makefile
@@ -457,44 +447,24 @@
 core-plugins/help_xts/Makefile
 core-plugins/help_bookmarks/Makefile
 
-dnl ***************************************************************************
-dnl fr: Script de compilation : Filtres
-dnl en: Compilation script : Filters
-dnl ***************************************************************************
 
 filters/gfx_pstricks/Makefile
 filters/gfx_svg/Makefile
 filters/text_csv/Makefile
 
-dnl ***************************************************************************
-dnl fr: Script de compilation : Greffons de données : Générale
-dnl en: Compilation script : Data plug-ins : General
-dnl ***************************************************************************
 
 data-plugins/general/Makefile
 data-plugins/general/tree/Makefile
 data-plugins/general/network/Makefile
 
-dnl ***************************************************************************
-dnl fr: Script de compilation : Greffons de données : Cogitation
-dnl en: Compilation script : Data plug-ins : Cogitation
-dnl ***************************************************************************
 
 data-plugins/cogitation/Makefile
 
-dnl ***************************************************************************
-dnl fr: Script de compilation : Greffons de données : Gestionnaire de projet
-dnl en: Compilation script : Data plug-ins : Project manager
-dnl ***************************************************************************
 
 data-plugins/project/Makefile
 data-plugins/project/gantt/Makefile
 data-plugins/project/schedule/Makefile
 
-dnl ***************************************************************************
-dnl fr: Script de compilation : Documentation
-dnl en: Compilation script : Documentation
-dnl ***************************************************************************
 
 doc/Makefile
 doc/draft/Makefile
